site/docs/components/slider.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Slider
+navTitle: Slider (not built)
+summaryParagraph: A slider lets people imprecisely select a value from a subjective range by moving a handle horizontally.
+tags: ["Range", "Range input", "Range slider"]
+needToKnow:
+- Avoid sliders when choosing a precise value is important.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### Examples
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D47508%253A35978&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Sliders are helpful for choosing subjective values.
+
+- Always show the current, selected value.
```

Comment:
I’m not fully sure if the new sliders show the current value but it sounds like this is something we should do (unless you mean just the visual “snap” to a value)